examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -99,6 +101,14 @@
                   {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
+                  },
+                  {
+                    "name": "KUBERNETES_MASTER",
```

Comment:
I don't consider the admission on off interface acceptable.  The enabled
flag was specifically put in to give all cluster admins a single,
explicable point of control for jenkins auto creation.  Admission fiddling
is not usable, discoverable, or safe.

On Wed, Aug 10, 2016 at 9:25 AM, Cesar Wong <notifications@github.com>
wrote:

> In examples/jenkins/jenkins-ephemeral-template.json
> <https://github.com/openshift/origin/pull/10260#discussion_r74242597>:
>
> > @@ -99,6 +101,14 @@
> >                    {
> >                      "name": "JENKINS_PASSWORD",
> >                      "value": "${JENKINS_PASSWORD}"
> > +                  },
> > +                  {
> > +                    "name": "KUBERNETES_MASTER",
>
> well, technically you could just change the run command in the pod without
> changing the image, but not sure it's worth the trouble.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10260/files/edcd36a6f3be520bad6588d34ec75dba5ffd9a9a#r74242597>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p4SfQnyBouRIxZ-6es9ymGlpYsuoks5qedE2gaJpZM4Jer_v>
> .
>